nixos/eva/modules/telegraf/default.nix

```diff
@@ -3,7 +3,6 @@
     ./private.nix
     ./uni.nix
     ./krebs.nix
-    ./nix-community.nix
```

Comment:
Should we not at least do meta-monitoring on the monitoring server itself?

Reply:
I've reverted the ssh check for web02.